ci/pipelines/geode-build/jinja.template.yml

```diff
@@ -89,8 +89,8 @@ GRADLE_GLOBAL_ARGS: ((gradle-global-args))
 {%- endfor -%}
 {% endmacro %}
 
-{% macro all_gating_except_windows_jobs() %}
-{%- for test in (tests) if not test.name=="StressNew" and test.PLATFORM=="linux" -%}
+{% macro all_gating_jobs_for_benchmark() %}
+{%- for test in (tests) if not test.name=="StressNew" and not test.name.startswith("Windows") -%}
```

Comment:
@onichols-pivotal , i disagree with your change request. Testing a name against a magic string is much less useful than using a token indicating the machine type. Specifically, Concourse already has a `platform` token for each task type, that corresponds to the worker type we are using. Our "heavy lifter" pattern is hiding our use of windows workers, or else this would be `double-plus-good`.

Reply:
@rhoughton-pivot Please re-read my request.  I only asked for consistency.  That could just as easily have been satisfied by changing the existing magic string checks to use the new attribute system.  If you would like to see that done, please submit an issue!